pkg/arvo/sys/zuse.hoon

```diff
@@ -2265,7 +2265,6 @@
           ~/  %sosi
           |=  [sk=@I m=@I a=@I]
           ^-  @J
-          ?>  (gte 32 (met 3 sk))
```

Comment:
It's important to be straightforward to verify that assertions/crashes in the jet match the hoon/nock. Can you add a comment to the source in addition to the commit msg that removed this?